src/Model/SalesOrder/SalesOrderItem.php

```diff
@@ -35,6 +35,13 @@ class SalesOrderItem
 	 * @var int|null
 	 */
 	private $taxCode;
+	/**
+	 * This is only used by Poslovi
+	 *
+	 * @SerializedName("custcol_rsm_package_quantity")
+	 * @var int|null
+	 */
+	private $posloviJobPackageQuantity;
```

Comment:
Woohoo, prvi slučaj namespace :1st_place_medal: 
Prvo mi je palo na pamet da li možda da `PosloviSalesOrderItem` extend-uje `SalesOrderItem`, ali to bi onda zahtevalo da postoji i `PosloviSalesOrderForm` koji extend-uje `SalesOrderForm` i onda apstrakcija k vragu..
Lično, sudeći po RSM nazivu, bliži mi je naziv property-ja `packageQuantity` a da u phpDoc stoji naznaka da se polje trenutno koristi kod Poslova (ko zna, možda ga još neko /zlo/upotrebi kasnije? :D )

Reply:
Da li ti je to pre-generički? @silvesterk

Reply:
Nah možemo tako, mislim  trenutno je zamišljeno da se koristi samo na poslovima, u pitanju je situacija gde želimo da naglasimo i pratimo u statistikama ako smo prodali npr: 1 x Paket od 15 Start Oglasa. Do sada smo to radili neispravno (jer nije postojao način da se uradi drugačije) pa je glasilo: 15 x Paket Start Oglasa.

E sada postoji verovatnoća da će tako nešto nekom drugom biznisu da bude potrebno u budućnosti. Ispravljam da bude generično a u komentaru ostavljam da to za sada samo poslovi koriste.